l1infotreesync/processor.go

```diff
@@ -123,16 +123,16 @@ func (l *L1InfoTreeInitial) String() string {
 }
 
 // Hash as expected by the tree
-func (l *L1InfoTreeLeaf) hash() common.Hash {
+func (l *L1InfoTreeLeaf) GetHash() common.Hash {
 	var res [treeTypes.DefaultHeight]byte
 	t := make([]byte, 8) //nolint:mnd
 	binary.BigEndian.PutUint64(t, l.Timestamp)
-	copy(res[:], keccak256.Hash(l.globalExitRoot().Bytes(), l.PreviousBlockHash.Bytes(), t))
+	copy(res[:], keccak256.Hash(l.GetGlobalExitRoot().Bytes(), l.PreviousBlockHash.Bytes(), t))
 	return res
 }
 
 // GlobalExitRoot returns the GER
-func (l *L1InfoTreeLeaf) globalExitRoot() common.Hash {
+func (l *L1InfoTreeLeaf) GetGlobalExitRoot() common.Hash {
```

Comment:
WDYT? Otherwise, update the comment, to conform to the updated function name (namely `GetGlobalExitRoot`)
```suggestion
func (l *L1InfoTreeLeaf) GlobalExitRoot() common.Hash {
```

Reply:
since  L1InfoTreeLeaf already has a field `GlobalExitRoot`, cant have the same name func

Reply:
Just update the function comment, to be `GetGlobalExitRoot` (consistent with the function name)